src/haddock/modules/analysis/seletopclusts/defaults.yaml

```diff
@@ -14,8 +14,8 @@ top_models:
   min: -9999
```

Comment:
Why -9999 is an allowed value ?
Maybe information in the `long` description to explain how to use negative values could be nice ? or should it be limitted to `min = 1` ?